webapp/src/mocks/api/mockPertubationAPI.ts

```diff
@@ -0,0 +1,471 @@
+            perturbedUtterance: "i need my bank account frozen.",
+          },
+        },
+        {
+          name: "Inner Comma",
+          description: "Delete comma inside the utterance.",
+          family: "Punctuation",
+          perturbationType: "Deletion",
+          evalFailureRate: 0.0,
+          evalCount: 1,
+          evalFailedCount: 0,
+          evalConfidenceDelta: 0.0034,
+          trainFailureRate: 0.0,
+          trainCount: 1,
+          trainFailedCount: 0,
+          trainConfidenceDelta: 0.0012,
+          example: {
+            utterance: "what is up with my account, why is it blocked",
+            perturbedUtterance: "what is up with my account why is it blocked",
+          },
+        },
+        {
+          name: "Inner Comma",
+          description: "Insert comma inside the utterance.",
+          family: "Punctuation",
+          perturbationType: "Insertion",
+          evalFailureRate: 0.13636363636363635,
+          evalCount: 22,
+          evalFailedCount: 3,
+          evalConfidenceDelta: 0.03607894736842105,
+          trainFailureRate: 0.0625,
+          trainCount: 16,
+          trainFailedCount: 1,
+          trainConfidenceDelta: 0.05998,
+          example: {
+            utterance: "i need my bank account frozen",
+            perturbedUtterance: "i need my, bank account frozen",
+          },
+        },
+        {
+          name: "Inner Period",
+          description: "Insert period inside the utterance.",
+          family: "Punctuation",
+          perturbationType: "Insertion",
+          evalFailureRate: 0.17391304347826086,
+          evalCount: 23,
+          evalFailedCount: 4,
+          evalConfidenceDelta: 0.032878947368421056,
+          trainFailureRate: 0.058823529411764705,
+          trainCount: 17,
+          trainFailedCount: 1,
+          trainConfidenceDelta: 0.06339375,
+          example: {
+            utterance: "i need my bank account frozen",
+            perturbedUtterance: "i need my. bank account frozen",
+          },
+        },
+        {
+          name: "Question Mark",
+          description: "Append question mark at the end of the utterance.",
+          family: "Punctuation",
+          perturbationType: "PostInsertion",
+          evalFailureRate: 0.08695652173913043,
+          evalCount: 23,
+          evalFailedCount: 2,
+          evalConfidenceDelta: 0.018895238095238093,
+          trainFailureRate: 0.11764705882352941,
+          trainCount: 17,
+          trainFailedCount: 2,
+          trainConfidenceDelta: 0.01038,
+          example: {
+            utterance: "i need my bank account frozen",
+            perturbedUtterance: "i need my bank account frozen?",
+          },
+        },
+      ],
+      failureRates: { eval: 0.14, train: 0.16015625 },
+    };
+    return res(ctx.json(datasetInfoResponse));
+  }
+);
+
+export const getPertubationResponseWithnoTrainFailureRate = rest.get(
```

Comment:
Detail, but I think `Withno` sould be `WithNo` or `Without`.
```suggestion
export const getPertubationResponseWithNoTrainFailureRate = rest.get(
```
or
```suggestion
export const getPertubationResponseWithoutTrainFailureRate = rest.get(
```

Reply:
okay I would go with this "getPertubationResponseWithoutTrainFailureRate"

Reply:
resolved